app/internal/comment.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+from typing import Dict, List
+
+from sqlalchemy.orm.session import Session
+
+from app.database.models import Comment, Event
+from app.internal import utils
+
+
+def create_comment(session: Session, event: Event, content: str) -> None:
+    """Creates a comment instance in the DB.
+
+    Args:
+        session (Session): DB session.
+        event (Event): Instance of the event to create a comment for.
+        content (str): The content of the comment.
+
+    Returns:
```

Comment:
I'd remove the parts of "Returns: None" / "Raises: None" to keep the code more easy to skim through